minimizer.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# Modified from https://github.com/google/fuzztest/blob/main/tools/minimizer.sh
+# If you want to use a specific reproducer, pass it in FUZZTEST_MINIMIZE_REPRODUCER
+# environment variable.
+# Otherwise, just run it as
+# $ ./minimizer.sh ./build/test/polygon_fuzz --fuzz=PolygonFuzz.TriangulationNoCrash
```

Comment:
Considering you already made a nice `minimize_testcase` function, should we just use that instead of the generic one gfuzz has?

Reply:
Probably not, `minimize_testcase` is mainly for geometrically valid polygons. Also, the way I wrote `minimize_testcase` is actually pretty similar to how fuzzer works, but not that general, so the fuzzer minimizer is strictly more powerful than our `minimize_testcase` code. We can probably remove the `minimize_testcase` code when the fuzzer integration is complete and easy to use.